3-java-servlet-web-app/1-Authentication/sign-in-ciam/README.md

```diff
@@ -0,0 +1,272 @@
+---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - msal-java
+  - microsoft-entra-id
+  - entra
+urlFragment: msal-java-authentication-sign-in-ciam
+description: "This sample demonstrates a Java Servlet web app that signs in CIAM users with the Microsoft identity platform"
+---
+
+# Enable your Java Servlet web app to sign in users to your Microsoft Entra tenant with the Microsoft identity platform
+
+- [Overview](#overview)
+- [Scenario](#scenario)
+- [Contents](#contents)
+- [Prerequisites](#prerequisites)
+- [Setup](#setup)
+  - [Clone or download this repository](#clone-or-download-this-repository)
+- [Register the sample application with your Microsoft Entra tenant](#register-the-sample-application-with-your-azure-active-directory-tenant)
+  - [Register the web app (java-servlet-webapp-authentication)](#register-the-web-app-java-servlet-webapp-authentication)
+  - [Configure the web app (java-servlet-webapp-authentication) to use your app registration](#configure-the-web-app-java-servlet-webapp-authentication-to-use-your-app-registration)
+- [Running the sample](#running-the-sample)
+- [Explore the sample](#explore-the-sample)
+- [We'd love your feedback!](#wed-love-your-feedback)
+- [About the code](#about-the-code)
+  - [Step-by-step walkthrough](#step-by-step-walkthrough)
+  - [Protecting the routes](#protecting-the-routes)
+  - [Scopes](#scopes)
+- [Next Steps or Deploy to Azure](#next-steps-or-deploy-to-azure)
+- [Community Help and Support](#community-help-and-support)
+- [Contributing](#contributing)
+- [Code of Conduct](#code-of-conduct)
+- [More information](#more-information)
+
+## Overview
+
+This sample demonstrates a Java Servlet web app that signs in users to your Microsoft Entra CIAM tenant using the [Microsoft Authentication Library (MSAL) for Java](https://github.com/AzureAD/microsoft-authentication-library-for-java).
+
+## Scenario
+
+1. This web application uses **MSAL for Java (MSAL4J)** to sign users into a Microsoft Entra CAIM tenant and obtain an [ID Token](https://docs.microsoft.com/azure/active-directory/develop/id-tokens) from **Microsoft Entra ID**.
```

Comment:
minor typo. "CAIM tenant" instead of CIAM

Reply:
Good catch, fixed in the latest commit